src/illuminator/models/Hotwaterstorage/hotwaterstorage_model.py

```diff
@@ -397,27 +490,65 @@ def get_nested_attr(self, nested_attr):
         return getattr(if_type[name], attr)
```

Comment:
For future reference. Exceptions are usually documented in doc strings under the heading `Raises`, with the format `<errorType>: <description of the error/exception>`.